CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,180 @@
+# OpenSSF Scorecard Contributor ladder
+
+This document outlines the various contributor roles in the Scorecard project, along with their respective prerequisites and responsibilities.
+It also defines the process by which users can request to change roles.
+
+- [Roles](#roles)
+  - [Community participants](#community-participants)
+  - [Community members](#community-members)
+  - [Triagers](#triagers)
+  - [Maintainers](#maintainers)
+- [Inactive members](#inactive-members)
+
+## Roles
+
+### Community participants
+
+Community participants engage with Scorecard,
+contributing their time and energy in discussions or just generally helping out.
+
+#### Pre-requisites
+
+- Must follow the [OpenSSF Code of Conduct]
+- Must follow the [Contribution Guide]
+
+#### Responsibilities
+
+- Keep it up!
+
+### Community Members
+
+Community Members are active contributors in the community.
+They can have issues and PRs assigned to them, participate through GitHub teams,
+and pre-submit tests are automatically run for their PRs.
+Members are expected to remain active contributors to the community.
+
+**Defined by:** Member of the OpenSSF GitHub organization
+
+#### Pre-requisites
+
+- Enabled two-factor authentication on their GitHub account
+- Have made multiple contributions to the project or community.
+  Contributions may include, but are not limited to:
+  - Authoring or reviewing PRs on GitHub. At least one PR must be **merged**.
+  - Filing or commenting on issues on GitHub
+  - Contributing to a project, or community discussions (e.g. meetings, Slack,
+    email discussion forums, Stack Overflow)
+- Active contributor to Scorecard or a relevant OpenSSF SIG
+
+#### Responsibilities
+
+- Can be assigned issues and PRs
+- Responsive to issues and PRs assigned to them
+- Others can ask for reviews with a `/cc @username`.
+- Responsive to mentions of teams they are members of
+- Active owner of code they have contributed (unless ownership is explicitly transferred)
+  - Ensures code is well tested and that tests consistently pass
+  - Addresses bugs or issues discovered after code is accepted
+
+#### Privileges
+
+- Tests run against their PRs automatically
+
+#### Promotion process
+
+- Sponsored by 2 maintainers or triagers. **Note the following requirements for sponsors**:
```

Comment:
What do you think about a maintainer or 2 triagers? As written, being a community member require more sponsorship than a triager.

Reply:
True! Fixed.